xlml/apis/task.py

```diff
@@ -320,6 +320,9 @@ class TpuXpkTask(BaseTask):
   task_test_config: test_config.TestConfig[test_config.Tpu]
   task_gcp_config: gcp_config.GCPConfig
   task_metric_config: Optional[metric_config.MetricConfig] = None
+  workload_provision_timeout: datetime.timedelta = datetime.timedelta(
```

Comment:
Do we want to set 5 hours as default? or 20hours? so users do not need to specify it

Reply:
Good point, let me increase it here. I think 5 hours is more reasonable, I was a bit overly ambitious to set 20 hours originally 😆